src/tests/conformance/query/ping_pong.rs

```diff
@@ -1,5 +1,5 @@
 //! Contains test with ping queries.
-//!
+//! ZG-CONFORMANCE-003
```

Comment:
I'm all for moving the spec test names somewhere above, but we should probably be consistent about it; @niklaslong?

Reply:
So long as the codebase is easily searchable and it is clear what cases the comment identifies in the file, I'm fine with either or even a combination.